tfsdklog/cli.go

```diff
@@ -0,0 +1,31 @@
+package tfsdklog
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/go-hclog"
+)
+
+const envLogCLI = "TF_LOG_CLI"
```

Comment:
Since this environment variable is very similarly named to its other `TF_LOG*` friends, it probably would be a really good idea to place an associated code comment here on what this does and what values it accepts. 🥇